js/qg-main/qg-utils.js

```diff
@@ -4,10 +4,55 @@
     // Workaround to push columns - some pages on Matrix have a 3 col layout but there is no third columns
     // example - https://www.qld.gov.au/about/events-awards-honours/awards/reconciliation-awards/2015-award-winners
     function colPush(){
-        let $qg = $('#qg-three-col');
+        var $qg = $('#qg-three-col');
         if ($qg.find('#qg-section-nav').length <= 0) {
             $qg.addClass('no-secondary-nav');
         }
     }
     colPush();
+    // This function equal height of cards in a group, if it finds a class '.qg-cards__equal-height and row classes i.e row-1 , row-2 etc'.
+    var $container = $('.qg-cards__equal-height');
+    var gridType = function (){
+        var fgridType;
+        if ($container.find('.col-lg-4').length > 0){
+            fgridType = 3;
+        } else if ($container.find('.col-lg-6').length > 0){
+            fgridType = 2;
+        }
```

Comment:
If those two conditions failed, the value won't be set at all; is that the desired behaviour?

Reply:
Practically this won't happen because the code is only triggered if we have a class .qg-cards__equal-height with the bootstrap grid of 2 or 3 columns.

Reply:
Is the second condition needed, then, if we expect it to always succeed?

Reply:
I have  changed the condition a bit to make default 3 cols per grid and added a note 
 // Equal height function only works with the Boostrap grid of 2 to 3 columns.